src/tests/ftest/osa/osa_online_reintegration.py

```diff
@@ -181,9 +102,10 @@ def run_online_reintegration_test(self, num_pool):
         rank = random.randint(1, exclude_servers)
 
         # Start the daos_racer thread
-        daos_racer_thread = threading.Thread(target=self.daos_racer_thread)
-        daos_racer_thread.start()
-        time.sleep(30)
+        if racer is True:
+            daos_racer_thread = threading.Thread(target=self.daos_racer_thread)
+            daos_racer_thread.start()
+            time.sleep(30)
```

Comment:
Same as above.

Reply:
daos_racer and TestPool creation definitely needs some time difference... If not TestPool creation was failing... Di's doesn't want to use daos_racer tool... This was the reason, I have to stub it out...